### PR TITLE
Fixing spelling mistake in README.md

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # durer_login
-Log in page for Durer competition.
+Login page for Durer competition.
```
Comment (merge25) [inline README.md]: Do you sure, it's written together?

Comment (mergerg) [reply README.md:2]: Yes, of course. Just look at it, it's obvious.

Comment (mergerg) [reply README.md]: Look now, it's even better.

Comment (gergo229) [reply README.md]: I see, see.`int main(){somecode}`

**Pull request merged**